### PR TITLE
⚡  Mt 969 header styling

### PR DESCRIPTION
**Pull request by miloszhygen**
[MT-969](https://itemtest.atlassian.net/browse/MT-969)

### CODE REVIEW

File: stories/Header.stories.svelte [hunk 1]
```diff
@@ -71,7 +71,7 @@
         />
       </svg>
     </a>
-    <Dropdown title="Språk/language" loadJs={true} let:titleId>
+    <Dropdown title="Språk/language" loadJs={true} buttonsClass="button__small-text" let:titleId>
```
Comment (lha047) [inline stories/Header.stories.svelte:74]:
```suggestion
    <Dropdown title="Språk/language" loadJs={true} class="button__small-text" let:titleId>
```

**Review by lha047 — CHANGES_REQUESTED** [review]
Looks good :D

Just have a look at my suggestions

File: src/svelte/Dropdown.svelte [hunk 2]
```diff
@@ -13,6 +13,7 @@
   const bodyId = `ui-dropdown-${counter++}`
   export let loadJs = false
   export let titleId = `${bodyId}-title`
+  export let buttonsClass = ''
```
[the remark this answers is not in the document]
Comment (miloszhygen) [reply src/svelte/Dropdown.svelte:16]: Good one 👍

**Review by lha047 — APPROVED** [review]
Nice :D

**Pull request merged**